source/xml/reading.md

```diff
@@ -376,6 +377,7 @@ their respective XML elements:
             }
             return $books;
         },
+        // handle a single book
         '{http://example.org/books}book' => function($reader) {
             $book = new Book();
```

Comment:
The namespace "ignorant" deserializer we talked about should just be a namespaced function, not a method on a subclass of `\Sabre\Xml\Element\`, right?